### ISSUE TITLE
more precise error message when invalid mailcap entry

### ISSUE THREAD

**Issue by teto** [pazz/alot#1371]
Before you submit a bug report, please make sure that the issue still exists on the master branch!

**Describe the bug**
When opening a certain message, I get:
`This message contains a text/html part that was not rendered due to a missing mailcap entry. Please refer to item 5 in our FAQ: http://alot.rtfd.io/en/latest/faq.html`

but when I look at the log:
```
DEBUG:utils:Content-Transfer-Encoding: "7bit"
DEBUG:utils:assuming Content-Transfer-Encoding: 7bit
DEBUG:utils:command: ${pkgs.w3m}/bin/w3m -dump -o document_charset=ISO-8859-1 '/tmp/zcgh5cgw.html'
DEBUG:utils:parms: ('text/html=', 'charset=ISO-8859-1')

```
it does find a mailcap entry, just launching the process failed. I would prefer to see a ui.notify(type=error, "couldn't launch command for mailcap text/html: <command>" instead of the current message that is incorrect (an entry exists, just it is not a valid command);

**Software Versions**
- Python version: 3.7
- Notmuch version: [if relevant]
- Alot version: 0.8

**To Reproduce**
put a wrong command for a specific mailcap entry

**Comment by lucc** [comment]
You are right, we should fix that. Sadly the check is currently made very late in `alot.db.message.Message.accumulate_body()` where we only have the empty rendered body and an original body that seems to be not empty.  We would have to refactor this to show a warning earlier.
